CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
 
 ___Note:__ Yet to be released changes appear here._
 
+* `DEPS`: bump @camunda/linting to v0.6.0 ([#](https://github.com/camunda/camunda-modeler/pull/))
```

Comment:
`([#](https://github.com/camunda/camunda-modeler/pull/))`

Which pull was it? 🐵

Reply:
Ooops. Mentioning the pull request you're about to create doesn't work so I added a placeholder and forgot to update. 🤡